### PR TITLE
[Doc] add content in troubleshooting mv

### PR DESCRIPTION
**Pull request by murphyatwork**
## What I'm doing:
- how to deal with refresh timeout
- how to use trace command to analyze the rewrite issue

Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function
- [ ] This is a backport pr

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.3
  - [x] 3.2
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5

### CODE REVIEW

File: docs/en/using_starrocks/troubleshooting_asynchronous_materialized_views.md [hunk 1]
```diff
@@ -245,23 +245,23 @@ If you failed to create an asynchronous materialized view, that is, the CREATE M
 
   In addition to the SQL statement, the main difference between the two materialized views is that asynchronous materialized views support all query syntax that StarRocks provides, but synchronous materialized views only support limited choices of aggregate functions.
 
-- **Check whether you have specified a correct  `Partition By`  column.**
+- **Check whether you have specified a correct `PARTITION BY` column.**
```
Comment (evelynzhaojie) [inline docs/en/using_starrocks/troubleshooting_asynchronous_materialized_views.md:248]: Check whether you have specified a correct partition column.

File: docs/zh/using_starrocks/troubleshooting_asynchronous_materialized_views.md [hunk 2]
```diff
@@ -270,32 +271,41 @@ MySQL > EXPLAIN LOGICAL SELECT `customer`.`c_custkey`
...
 
-  较大的物化视图可能因为刷新任务超过超时时间而无法刷新。要解决这个问题，您可以：
+较大的物化视图可能因为刷新任务超过超时时间而无法刷新，通常有以下几种解决方案。
 
-  - 为物化视图指定分区策略，实现细粒度的刷新。
-  - 增加超时时间。
+- **为物化视图指定分区策略，实现细粒度的刷新**
 
-从 v3.0 版本开始，您可以在创建物化视图时定义以下属性（Session Variable），或者使用 ALTER MATERIALIZED VIEW 命令进行添加：
+  如 [创建分区物化视图](./create_partitioned_materialized_view.md) 所描述，对物化视图进行可以分区可以实现增量构建与刷新，能够规避在初始刷新时占用太多资源的问题。
```
Comment (evelynzhaojie) [inline docs/zh/using_starrocks/troubleshooting_asynchronous_materialized_views.md:291]: 对物化视图进行可以分区 -- 删除 可以

File: docs/zh/using_starrocks/troubleshooting_asynchronous_materialized_views.md [hunk 3]
```diff
@@ -270,32 +271,41 @@ MySQL > EXPLAIN LOGICAL SELECT `customer`.`c_custkey`
...
+  ```SQL
+  ALTER MATERIALIZED VIEW mv2 SET ( 'session.query_timeout' = '4000' );
+  ```
+
+- **分析物化视图性能瓶颈**
+
+  如果物化视图计算复杂，其本身计算耗时就会很久。您可以通过 Query Profile 分析性能瓶颈，并进行优化：
+
+  1. 通过查询 `information_schema.task_runs` 获取刷新任务的 `query_id`。
+  2. 通过上述的 `query_id`，获取并分析分析其 Query Profile。
```
Comment (evelynzhaojie) [inline docs/zh/using_starrocks/troubleshooting_asynchronous_materialized_views.md:306]: 分析分析 -- 重复

File: docs/en/using_starrocks/troubleshooting_asynchronous_materialized_views.md [hunk 4]
```diff
@@ -270,32 +270,41 @@ If the materialized view fails to refresh, that is, the state of the refresh tas
...
-Example:
+- **Set a longer timeout period**
 
-```SQL
--- Define the properties when creating the materialized view
-CREATE MATERIALIZED VIEW mv1 
-REFRESH ASYNC
-PROPERTIES ( 'session.enable_spill'='true' )
-AS <query>;
+  The default timeout for materialized view refresh tasks is 5 minutes versions earlier than v3.2 and 1 hour in v3.2 and later. If you encounter timeout exceptions, you can adjust the timeout period by using the following statement:
```
Comment (evelynzhaojie) [inline docs/en/using_starrocks/troubleshooting_asynchronous_materialized_views.md:294]: versions earlier than v3.2 --- in versions earlier than v3.2

File: docs/en/using_starrocks/troubleshooting_asynchronous_materialized_views.md [hunk 5]
```diff
@@ -270,32 +270,41 @@ If the materialized view fails to refresh, that is, the state of the refresh tas
...
-ALTER MATERIALIZED VIEW mv2 
-    SET ('session.enable_spill' = 'true');
-```
+  ```sql
+  ALTER MATERIALIZED VIEW mv2 SET ('session.query_timeout' = '4000');
+  ```
+
+- **Analyze performance bottlenecks of the materialized view refresh**
+
+  Refreshing materialized views with complex computation is inherently time-consuming. You can analyze its performance bottlenecks by analyzing the query profile of the refresh task:
```
Comment (evelynzhaojie) [inline docs/en/using_starrocks/troubleshooting_asynchronous_materialized_views.md:302]: remove ”inherently“

File: docs/en/using_starrocks/troubleshooting_asynchronous_materialized_views.md [hunk 6]
```diff
@@ -311,15 +320,15 @@ If setting the materialized view state to active does not take effect, you need
 
 If you find that the refresh tasks are using excessive system resources, you can look into the following aspects:
 
-- **Check whether you have created an oversized  materialized view****.**
+- **Check whether you have created an oversized materialized view.**
```
Comment (evelynzhaojie) [inline docs/en/using_starrocks/troubleshooting_asynchronous_materialized_views.md:323]: Check whether you have created a materialized view too large.

File: docs/en/using_starrocks/troubleshooting_asynchronous_materialized_views.md [hunk 7]
```diff
@@ -331,43 +340,60 @@ To stop a refresh task that occupies too many resources, you can:
...
 
-### Materialized view fails to rewrite queries
+### Materialized view query rewrite failure
 
 If your materialized view fails to rewrite relevant queries, you can look into the following aspects:
 
-- **Check whether the  materialized view  and the  query  match.**
+- **Diagnose rewrite failure using TRACE**
+
+  StarRocks supports the TRACE statement for the diagnosis of materialized view rewrite failure:
```
Comment (evelynzhaojie) [inline docs/en/using_starrocks/troubleshooting_asynchronous_materialized_views.md:355]: StarRocks supports using the TRACE statement to diagnose materialized view rewrite failure:

File: docs/en/using_starrocks/troubleshooting_asynchronous_materialized_views.md [hunk 8]
```diff
@@ -331,43 +340,60 @@ To stop a refresh task that occupies too many resources, you can:
...
+  
+    - `TRACE LOGS MV <query>`: Available in v3.2 and later, this command analyzes the detailed rewrite process and the reasons of the failure.
+    - `TRACE REASON MV <query>`: Available in v3.2.8 and later, this command provides concise reasons for rewrite failure.
```
Comment (evelynzhaojie) [inline docs/en/using_starrocks/troubleshooting_asynchronous_materialized_views.md:358]: this command provides concise for rewrite failure reasons .

**Review by EsoragotoSpirit — APPROVED** [review]

**Review by evelynzhaojie — APPROVED** [review]

**Pull request merged**